tests/unit/setup.js

```diff
@@ -14,3 +14,4 @@ window.ResizeObserver ??= ResizeObserverStub
 // Mock element scroll API as not currently included in jsdom:
 // https://github.com/jsdom/jsdom/issues/1422
 Element.prototype.scrollBy ??= function () { }
+Element.prototype.scrollIntoView ??= function () { }
```

Comment:
Ok, might as well put the whole lot in to avoid potential future pain for devs:
```suggestion
Element.prototype.scrollIntoView ??= function () { }
Element.prototype.scroll ??= function () { }
Element.prototype.scrollTo ??= function () { }
```

Reply:
Although I'm curious as to why this is needed on this branch?